src/WorkshopType.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace PhpSchool\PhpWorkshop;
+
+use MyCLabs\Enum\Enum;
+
+/**
+ * @method static WorkshopType STANDARD()
+ * @method static WorkshopType TUTORIAL()
+ * @author Michael Woodward <mikeymike.mw@gmail.com>
+ */
+class WorkshopType extends Enum
+{
+    const STANDARD = 1;
+    const TUTORIAL = 2;
```

Comment:
You can actually add methods here if you wanted like `isTutorialMode()` then you can just call that up above